src/module-elasticsuite-virtual-category/Model/Layer/Filter/Category.php

```diff
@@ -56,6 +64,7 @@ public function __construct(
         \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
         \Smile\ElasticsuiteCore\Api\Search\ContextInterface $context,
         \Smile\ElasticsuiteVirtualCategory\Model\Category\Filter\Provider $filterProvider,
+        RequestFieldMapper $requestFieldMapper,
```

Comment:
This should also be injected into the parent::__construct() call just below, shouldn't it ?

Reply:
Agreed